proto/v1/structure/question.proto

```diff
@@ -0,0 +1,28 @@
+syntax = "proto3";
+
+package org.oppia.proto.v1.structure;
+
+import "proto/v1/structure/image.proto";
+import "proto/v1/structure/state.proto";
+import "proto/v1/structure/versions.proto";
+
+option java_package = "org.oppia.proto.v1.structure";
+option java_multiple_files = true;
+
+// Structure of the single question. 
+message Question {
+  QuestionStructureVersion structure_version = 1;
+
+
+  // Id of the question.
+  string id = 2;
+
+  // Question state is the name of the state to which this question belongs.
```

Comment:
The State object that makes up the main part of the question.

Reply:
done